templates/workspaces/airlock-import-review/terraform/import_review_resources.terraform

```diff
@@ -0,0 +1,41 @@
+# This file has a .terraform file extension in order to avoid 'terraform init's validation checks that are executed by the 'make bundle-build' command.
+# The Dockerfile includes a RUN command to change the extension from .terraform to .tf after the files from the base workspace are copied to this directory.
```

Comment:
as the comment explains, this terraform file is not complete as is, it requires the files from the base workspace template in order to pass the validations of `terraform init`